roles/edpm_neutron_dhcp/templates/wrappers/dnsmasq.j2

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+{% if edpm_neutron_dhcp_sidecar_debug -%}
+set -x
+{%- endif -%}
+
+ARGS="$@"
+
+IMAGE_NAME={{ edpm_neutron_dhcp_sidecar_dnsmasq_image_name }}
+
+{% raw -%}
+# Extract the network namespace UUID from the command line args provided by
+# neutron. Typically of the form (with dnsmasq as an example):
+#
+# dnsmasq --no-hosts --no-resolv --except-interface=lo \
+#   --pid-file=/var/lib/neutron/dhcp/317716b8-919a-4a6f-8db1-78128ec3b100/pid  \
+#   --dhcp-hostsfile=/var/lib/neutron/dhcp/317716b8-919a-4a6f-8db1-78128ec3b100/host ...
+NETNS=$(ip netns identify)
+NAME=neutron-dnsmasq-${NETNS}
+
+CLI="nsenter --net=/run/netns/${NETNS} --preserve-credentials -m -t 1 podman"
+LOGGING="--log-driver k8s-file --log-opt path=/var/log/containers/stdouts/${NAME}.log"
+CMD='/usr/sbin/dnsmasq -k'
+
+LIST=$($CLI ps -a --filter name=neutron-dnsmasq- --format '{{.ID}}:{{.Names}}:{{.Status}}' | awk '{print $1}')
+
+# Find orphaned containers left for dead after its main process terminated by neutron parent process
+# FIXME(cjeanner): https://github.com/containers/libpod/issues/1703
+ORPHANTS=$(printf "%s\n" "${LIST}" | grep -E ":(Exited|Created)")
+if [ -n "${ORPHANTS}" ]; then
+    for orphant in $(printf "%s\n" "${ORPHANTS}" | awk -F':' '{print $1}'); do
+        echo "Removing orphaned container ${orphant}"
+        $CLI stop ${orphant} || true
```

Comment:
These will pass no matter what, even if the runtime fails for some reason. It may be a good idea to at least report that the container wasn't removed.

Reply:
I don't want to change this logic now. It's like it was in Tripleo until now. I added TODO with Your comment about logging and I will want to get back to it later in the follow up PR. Would that be ok for You?

Reply:
Leaving it for later is fine. As long as we have a note about the deficiency.